src/components/filter/explicitNaming/ExplicitNamingFilterEditionDialog.tsx

```diff
@@ -35,7 +35,7 @@ const formSchema = yup
     .required();
 
 export interface ExplicitNamingFilterEditionDialogProps {
-    id: string;
+    id: string | null;
```

Comment:
`id` can't be null for `saveExplicitNamingFilter()` no?

Reply:
Same remark : 

currentExplicitNamingFilterId can be null in gridexplore-app, in directory-content.tsx :

<ExplicitNamingFilterEditionDialog
   id={currentExplicitNamingFilterId}